pandas/core/groupby/groupby.py

```diff
@@ -1102,14 +1102,11 @@ def _wrap_applied_output(self, data, keys, values, not_indexed_same: bool = Fals
     def _resolve_numeric_only(self, numeric_only: bool | lib.NoDefault) -> bool:
         """
         Determine subclass-specific default value for 'numeric_only'.
-
         For SeriesGroupBy we want the default to be False (to match Series behavior).
         For DataFrameGroupBy we want it to be True (for backwards-compat).
-
         Parameters
         ----------
         numeric_only : bool or lib.no_default
-
```

Comment:
and this

Reply:
I"ve done this in https://github.com/pandas-dev/pandas/pull/43154/commits/45f54d6cb548828072fc2d5885687d4d4db5b712